v2/protocol/http/protocol.go

```diff
@@ -24,7 +24,6 @@ const (
 type Protocol struct {
 	Target          *url.URL
 	RequestTemplate *http.Request
-	transformers    binding.Transformers
```

Comment:
don't you want some static ones set? or they should always come from the outside?